Agents/SewageAgent/SewerageNetworkAgent/pom.xml

```diff
@@ -0,0 +1,109 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>uk.ac.cam.cares.jps</groupId>
+  <artifactId>seweragenetwork-agent</artifactId>
+  <version>1.0.0</version>
+  <packaging>war</packaging>
+	<!-- Parent POM -->
+	<parent>
+		<groupId>uk.ac.cam.cares.jps</groupId>
+		<artifactId>jps-parent-pom</artifactId>
+		<version>1.0.0</version>
```

Comment:
Please use something more recent, e.g. 2.0.0.

Reply:
I have done all the amendments according to the comments. Thank you very much for the feedback and please let me know any further changes that are required.